doc/introduction/chemistry.rst

```diff
@@ -21,38 +21,38 @@ Eigensolver (VQE) algorithm.
 Building the electronic Hamiltonian
 -----------------------------------
 
-The ``qchem`` module provides access to a driver function :func:`~.generate_hamiltonian`
+The ``qchem`` module provides access to a driver function :func:`~.molecular_hamiltonian`
 to generate the electronic Hamiltonian in a single call. For example,
 
 .. code-block:: python
 
-    h, nr_qubits = qml.qchem.generate_hamiltonian(
+    h, n_qubits = qml.qchem.molecular_hamiltonian(
         name='h2',
         geo_file='h2.xyz',
         charge=0,
-        multiplicity=1,
-        basis_set='sto-3g',
-        n_active_electrons=2,
-        n_active_orbitals=2
+        mult=1,
+        basis='sto-3g',
+        nact_els=2,
```

Comment:
I really don't like `nact_els` and `nact_orbs`. Why not just `active_electrons = 2` and `active_orbitals = 2`? I think this is clear and simple

Reply:
Agree 💯

Reply:
I agree with the long names. The prefix "n" is important to make the distinction "number of ..", otherwise it may be difficult for the user to differentiate at glance if `active_orbitals` refers to the number of active orbitals or to an array with the indices of the active orbitals. This is kind of standard in OpenFermion as well, indeed, this is coming from the input style of qchem packages. This way it is easier to educate the user that the prefix "n" is always an integer value referring to a "number of ...", thoughts ?

Reply:
I tend to disagree; if I see `active_electrons [int]:` in the documentation, the `int` indicates to me that this refers to a 'number of...'.

We use this is a lot in PennyLane; sometimes `wires=` is the number of wires, sometimes it is a list of wires:

```python
dev = qml.dev("default.qubit", wires=2)
obs = qml.Hermitian(A, wires=[0, 3])
```

But I can't speak too much for the standard in qchem!

Reply:
That is true. But for example, the function `active_space` returns precisely the indices of the core and active orbitals. Imagine having something like this:

```
core_orbitals, active_orbitals = active_space(n_electrons, n_orbitals, active_electrons=2, active_orbitals=2)
```
This looks confusing to me but if this is OK with most of you, democracy now!.

Reply:
@agran2018 your code example definitely makes me 🤔 !

I'm going to vote to abstain 😆

Reply:
It seems that there are two different properties that should be named: the "number" of active electrons and the "indices" of active electrons. Maybe one can use `active_electrons` for the number and `electron_indices` for the indices. I think the problem is to find two good names. Same applies to the active orbitals.

Reply:
Thanks @soranjh. This sounds even more confusing to me as we don't assign indices to electrons but to orbitals. In general taking about electrons/particles without "n" is weird/counter-intuitive. Let's give it a try and let's see how it looks.

Reply:
Correct, I meant orbital indices.